cmd/kube-apiserver/apiserver.go

```diff
@@ -42,6 +45,7 @@ func main() {
 	defer logs.FlushLogs()
 
 	if err := command.Execute(); err != nil {
-		os.Exit(1)
+		exitCode = 1
+		return
```

Comment:
Not sure I understand why we need a return here since we're about to finish the execution of main().